src/keri/app/cli/commands/escrow/list.py

```diff
@@ -0,0 +1,279 @@
+# -*- encoding: utf-8 -*-
+"""
+KERI
+keri.kli.commands.escrow module
+
+"""
+import argparse
+import json
+
+from hio import help
+from hio.base import doing
+
+from keri.core import eventing
+from keri.app.cli.common import existing
+from keri.db import dbing
+from keri.kering import ConfigurationError
+from keri.vdr import viring
+
+logger = help.ogler.getLogger()
+
+parser = argparse.ArgumentParser(description='Views events in escrow state.')
+parser.set_defaults(handler=lambda args: handler(args),
+                    transferable=True)
+parser.add_argument('--name', '-n', help='keystore name and file location of KERI keystore', required=True)
+parser.add_argument('--base', '-b', help='additional optional prefix to file location of KERI keystore',
+                    required=False, default="")
+parser.add_argument('--passcode', '-p', help='21 character encryption passcode for keystore (is not saved)',
+                    dest="bran", default=None)  # passcode => bran
+
+parser.add_argument("--escrow", "-e", help="show values for one specific escrow", default=None)
+
+
+def handler(args):
+    """ Command line escrow handler
+
+    """
+    kwa = dict(args=args)
+    return [doing.doify(escrows, **kwa)]
+
+
+def escrows(tymth, tock=0.0, **opts):
+    _ = (yield tock)
+
+    args = opts["args"]
+    name = args.name
+    base = args.base
+    bran = args.bran
+    escrow = args.escrow
+
+    try:
+        with existing.existingHby(name=name, base=base, bran=bran) as hby:
+            escrows = dict()
+
+            # KEL / Baser escrows
+
+            if (not escrow) or escrow == "unverified-receipts":
+                count = 0
```

Comment:
Also good is `sum(1 for k, v in <iter>)` in python if you want to save some lines.

Reply:
Yes, I do want to save some lines. I was thinking there was a reduction function I could use like this, so thanks. This has been addressed in the new code